selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java

```diff
@@ -237,18 +181,58 @@ private Locators() {}
     }
   }
 
-  @FindBy(id = Locators.EDITOR_TABS_PANEL)
-  private WebElement editorTabsPanel;
+  /** Editor`s markers types */
+  public enum MarkersType {
+    ERROR_MARKER_OVERVIEW("//div[@class='ruler annotations']/div[@class='annotation error']"),
+    WARNING_MARKER_OVERVIEW("//div[@class='ruler annotations']/div[@class='annotation warning']"),
+    TASK_MARKER_OVERVIEW("//div[@class='ruler annotations']/div[@class='annotation task']"),
+    ERROR_MARKER("//div[@class='ruler overview']/div[@class='annotationOverview error']"),
+    WARNING_MARKER("//div[@class='ruler overview']/div[@class='annotationOverview warning']"),
+    INFO_MARKER("//div[@class='annotationHTML info']");
+
+    private final String itemLocator;
```

Comment:
What about returning `By locator`?

Reply:
In my opinion, it should be "String". It gives us flexible. Because locator can be of the different types (By.id, By.name, By.xpath, By.css and other). And "String" can contain all of them.